src/tests/functional/plugin/shared/include/behavior/ov_executable_network/exec_network_base.hpp

```diff
@@ -102,14 +104,14 @@ TEST_P(OVExecutableNetworkBaseTest, checkGetExecGraphInfoIsNotNullptr) {
 
 TEST_P(OVExecutableNetworkBaseTest, checkGetMetric) {
     auto execNet = core->compile_model(function, targetDevice, configuration);
-    EXPECT_NO_THROW(execNet.get_metric(METRIC_KEY(SUPPORTED_CONFIG_KEYS)));
+    EXPECT_NO_THROW(execNet.get_property(METRIC_KEY(SUPPORTED_CONFIG_KEYS)));
```

Comment:
why don't we use new typed properties here?

Reply:
there is no supported config keys typed key it is needed to rewrite the code using `ov::supported_metric`. Going to do it in a separate PR.